tests/GetUniqueFirstLettersTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+class GetUniqueFirstLettersTest extends TestCase
+{
+    /**
+     * @dataProvider providerPositiveData
+     */
+    public function testPositive($input, $expected)
```

Comment:
please add more test for all functions and provide some negative test cases if possible